cmd/buildah/run.go

```diff
@@ -137,14 +137,14 @@ func runCmd(c *cobra.Command, args []string, iopts runInputOptions) error {
 		Isolation:        isolation,
 		NamespaceOptions: namespaceOptions,
 		ConfigureNetwork: networkPolicy,
-		ContextDir:       iopts.contextDir,
 		CNIPluginPath:    iopts.CNIPlugInPath,
 		CNIConfigDir:     iopts.CNIConfigDir,
 		AddCapabilities:  iopts.capAdd,
 		DropCapabilities: iopts.capDrop,
 		Env:              iopts.env,
 		WorkingDir:       iopts.workingDir,
 	}
+	options.RunMountSources.ContextDir = iopts.contextDir
```

Comment:
Go ahead and include this in the initializer, as the hunk for `imagebuildah/stage_executor.go` does.